buildkite/benchmark/run.py

```diff
@@ -50,6 +53,53 @@
             "ARROW_SRC": f"{build_dir}/arrow",  # required by Java Script benchmarks
         },
     },
+    {
+        "benchmarkable_type": "arrow-commit",
+        "repo": "https://github.com/voltrondata-labs/arrowbench.git",
+        "root": "arrowbench",
+        "branch": "main",
+        "setup_commands": [],
+        "path_to_benchmark_groups_list_json": "arrowbench/inst/benchmarks.json",
+        "url_for_benchmark_groups_list_json": "https://raw.githubusercontent.com/voltrondata-labs/arrowbench/main/inst/benchmarks.json",
+        "setup_commands_for_lang_benchmarks": {  # These commands need to be defined as functions in buildkite/benchmark/utils.sh
+            "R": [
+                "build_arrow_r",
+                "install_arrowbench",
+                "create_data_dir",
+            ],
+        },
+        "env_vars": {
+            "PYTHONFAULTHANDLER": "1",  # makes it easy to debug segmentation faults
```

Comment:
Do we need this here? Or does it get passed all the way down to the adapter and we want those seg faults boiled up?

Reply:
This is left over from when I copied it over from the labs/benchmarks one. I think it might get passed through all the way to adapters, but if it matters at that level I'd rather set it at that level (as doing so is easy). I'll remove this.